misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeAction.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein is strictly forbidden, unless permitted by WSO2 in accordance with
+ * the WSO2 Commercial License available at http://wso2.com/licenses.
+ * For specific language governing the permissions and limitations under
+ * this license, please see the license as well as any agreement you’ve
+ * entered into with WSO2 governing the purchase of this software and any
+ */
+package org.ballerinalang.datamapper;
```

Comment:
It seems the license header is not properly added. The last part is missing and also new lines are missing. Refer [1] for example.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/preloading/LSClientConfig.java

Reply:
Changed the license header as suggested